devito/tools/abc.py

```diff
@@ -117,6 +117,10 @@ class Pickable(object):
     _pickle_kwargs = []
     """The keyword arguments that need to be passed to __new__ upon unpickling."""
 
+    @staticmethod
+    def wrapper(cls, args, kwargs):
```

Comment:
this should be private (ie prefix with `_`), and perhaps be called `_pickle_wrapper`. Since `Pickable` is inherited by our first-class citizen classes (eg, `Function`), I think this is a fairly useful tweak